internal/cli/update.go

```
@@ -0,0 +1,104 @@
+		if result.Latest == "" {
+			// A catalog with no releases at all means this build is not a
+			// release (a dev build), not that it is current.
+			if len(result.Releases) == 0 {
+				return errors.New("update: this build is not a release; reinstall to upgrade")
+			}
+			return emit(out, *asJSON, updateReport{UpToDate: true}, "already up to date")
+		}
+		// The download can take a while, so say what is coming; --json keeps
+		// stdout to the single record.
+		if !*asJSON {
+			fmt.Fprintf(out, "updating to %s...\n", result.Latest)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)